src/client/resources/conversions.ts

```diff
@@ -282,7 +282,7 @@ export function createMigMigration(
       // TODO: We need to wire these up to the UI when stage is implemented
       // and we've more thouroughly tested quiesce/stage combos in conjunction
       // with the controller and velero.
-      quiescePods: true,
+      quiescePods: !isStage,
```

Comment:
This is tricky, and it's not strictly a function of whether or not we're doing a stage. During a migration, the user is presented an option to quiesce or not. I would remove this from the PR and hit later in a distinct PR, since it's more nuanced. See: https://github.com/fusor/mig-ui/issues/224

Reply:
Ok, I'll try to implement check box there later. Thanks for the suggestion, the issue https://github.com/fusor/mig-ui/issues/323 implied that it should be connected directly to the `stage` logic for now.